README.md

```diff
@@ -8,18 +8,6 @@
 
 ## Compatibility
 
-------------------------------------------------------------------------------
-
-***NOTE***
-
-  The addon's dummy application needs a serious upgrade.  I [@tylerturdenpants](https://github.com/tylerturdenpants) have removed as much orf the support on flexi and flexi styles that I can, but the dummy app will still produces warnings regarding SASS division.
-
-  Additionally, this app likely has support beyond ember 3.24, but test fail locally since not all babel related issues during testing have been resolved.
```

Comment:
I think we can remove this part exclusively as the remaining dummy-app-related text is still relevant.